components/Rank/__tests__/index.js

```diff
@@ -0,0 +1,27 @@
+import {mount} from 'enzyme'
+import React from 'react'
+import Rank from '../'
+
+describe('Rank', () => {
+  beforeEach(() => {
+    console.error = jest.fn((error) => {
```

Comment:
My thoughts on this:

We could use Sinon to stub and restore the function:

```js
// stub
sinon.stub(console, 'error', warning => { throw new Error(warning) })

// restore
console.error.restore()
```

We could use Jest [Setup and Teardown](https://facebook.github.io/jest/docs/setup-teardown.html) functions to run this before and after each test.

Additionally, if we wanted to specify this on a global level, we could put it in a global setup file, such as `test/setup.js`, and require it before each test by putting this in `package.json`:

```json
"jest": {
  "setupFiles": [
    "./test/setup.js"
  ]
}
```

The downside of this is any test in the suite will fail if any `error` is logged. This demands that all dependencies use `console.error` semantically, which many not be the case.